app/views/unlocalized_words.php

```diff
@@ -0,0 +1,28 @@
+<?php
+namespace Transvision;
+
+// Include the common simple search form
+include __DIR__ . '/simplesearchform.php';
+
+$search_id = 'unlocalized_strings';
+
+$content = "<table class='collapsable results_table {$search_id}'>\n" .
+            "  <tr class='column_headers'>\n" .
+            "    <th>English</th>\n" .
+            "    <th>Occurrences</th>\n" .
+            "    <th></th>\n" .
+            "  </tr>\n";
+
+foreach ($unlocalized_words as $string_id => $string_value) {
+    $link = "/?recherche={$string_id}&repo={$repo}&sourcelocale=en-US" .
+            "&locale={$locale}&search_type=strings&whole_word=whole_word";
+
+    $content .= "  <tr class='{$search_id}'>\n" .
+                "    <td>{$string_id}</td>\n" .
+                "    <td>{$string_value}</td>\n" .
+                "    <td><a href='{$link}'>🔍</a></td>\n" .
```

Comment:
It probably makes sense to have some text here for accessibility, and eventually hide it with CSS.